vignettes/intro.Rmd

```diff
@@ -0,0 +1,162 @@
+---
+title: "A Brief Introduction to RNeXML"
+author:
+  - Carl Boettiger
+  - Scott Chamberlain
+  - Rutger Vos
+  - Hilmar Lapp
+
+output: html_vignette
+---
+
+<!--
+%\VignetteEngine{knitr::rmarkdown}
+%\VignetteIndexEntry{A Brief Introduction to RNeXMLL}
+-->
+
+
+```{r echo=FALSE}
+library(RNeXML)
+```
+
+Read in a `nexml` file into the `ape::phylo` format:
+
+```{r}
+f <- system.file("examples", "comp_analysis.xml", package="RNeXML")
+nexml <- nexml_read(f)
+tr <- get_trees(nexml) # or: as(nexml, "phylo")
+plot(tr)
+```
+
+Write an `ape::phylo` tree into the `nexml` format:
+
+```{r}
+data(bird.orders)
+nexml_write(bird.orders, "test.xml")
```

Comment:
```suggestion
nexml_write(bird.orders, "test.xml", creator = "Carl Boettiger")
```
This makes the result independent on machine and user. (The default is the username on the system.)